### PR TITLE
DR-775:Remove large commented-out areas without explanation

### PR DESCRIPTION
**Pull request by NamrataSweya**

### CODE REVIEW

File: apps/drec-api/migrations/9999999999999-Seed.ts [hunk 1]
```diff
@@ -72,179 +69,6 @@ export class Seed9999999999999 implements MigrationInterface {
 
   public async down(queryRunner: QueryRunner): Promise<any> {}
 
-  private async seedUsers(queryRunner: QueryRunner) {
```
Comment (dmohns) [inline apps/drec-api/migrations/9999999999999-Seed.ts:75]: Why is this removed? I was under the impression this does something, but maybe I'm wrong.

**Review by dmohns — COMMENTED** [review]
Nice, really good to see this old stuff getting removed 💪 🔥 🧹 

Left one comment/doubt. Other than that LGTM

File: apps/drec-api/migrations/9999999999999-Seed.ts (hunk 1 above)
Comment (NamrataSweya) [reply apps/drec-api/migrations/9999999999999-Seed.ts:75]: yes this is also use for adding user ,organization and device from using static json file to add data for testing ,but i thinking ,@Aiswarya also make same way for test so may be now its not require.

Comment (dmohns) [reply apps/drec-api/migrations/9999999999999-Seed.ts:75]: Not sure I understand this. @Aish1990 could you confirm?

Comment (Aish1990) [reply apps/drec-api/migrations/9999999999999-Seed.ts:75]: Sir, Currently this function calls are already commented which we are not using... 
image
I think it's may be used to test migrations with static data at initial stage...

Comment (dmohns) [reply apps/drec-api/migrations/9999999999999-Seed.ts:75]: Ohh, I see.

**Review by dmohns — APPROVED** [review]
